src/ui/public/vislib/lib/axis/axis_config.js

```diff
@@ -1,6 +1,6 @@
 import _ from 'lodash';
 import d3 from 'd3';
-import { SCALE_MODES } from './scale_modes';
+import { SCALE_MODES } from '../../../vis/scale_modes';
```

Comment:
scale modes is something thats really vislib specific, should not be used from outside, and should not be part of vis